core-plugins/docs/LogParser-transform.md

```diff
@@ -21,6 +21,10 @@ Properties
 **inputName:** Name of the field in the input schema which encodes the
 log information. The given field must be of type ``String`` or ``Bytes``.
 
+Conditions
+----------
+If error dataset is configured then all the erroneous rows, if present in the input, will be committed to the
+specified error dataset. If not configured, the erroneous rows will be committed to default error dataset.
```

Comment:
If nothing is specified, it actually doesn't write to any dataset and logs a warning in the logs. There is no "default" error dataset.